pkg/test/server.go

```diff
@@ -57,7 +62,17 @@ func RunManagementServer(ctx context.Context, srv server.Server, port uint) {
 	// a single connection to the management server, then it might lead to
 	// availability problems.
```

Comment:
Sorry one more thing, under this comment could you maybe add a note regarding the envoy docs mentioning the keepalive settings? Same with `internal/example` code as well. This should be good to go after this

Reply:
Looks good, please review my changes.